pkg/server/container_create.go

```diff
@@ -225,6 +226,18 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	if username := securityContext.GetRunAsUsername(); username != "" {
 		specOpts = append(specOpts, oci.WithUsername(username))
 	}
+	if gid := securityContext.GetRunAsGroup(); gid != nil {
```

Comment:
FYI... oci.WithUserID() and oci.WithUsername() are already looking up the gid and may already be setting it. Course a user can be in more than one group.. so I'm not sure how well the current assignment is working. Thoughts?

Reply:
We set GID after `WithUserID` and `WithUsername` which overwrites the gid.

Reply:
per discussion let's todo a helper that does uid,username, and gid at the same time..